### DISCUSSION TITLE
Interest in a monthly meeting to discuss status / plans for Certifier Framework library.

### DISCUSSION THREAD

**Discussion by gapisback** [ccc-certifier-framework/certifier-framework-for-confidential-computing#239] [Polls]
Dear colleagues, 

Now that the dust has settled on the movement of this project to under CCC and the repo is under `ccc-certifier-framework/`, I want to get an idea of the interest level from the community members on a monthly meeting around this project.

I expect @jlmucb and maybe @rgerganov may be interested / able to attend. I will poll other people who have worked on this project off-line via email. 

We used to have a weekly meeting on Wed at 8 am PT. I think that time sort-of worked for most people on that project team. (Unfortuntately, this time won't work for Samsung colleagues. Not sure how to accomodate that time zone.)

See: https://www.timeanddate.com/worldclock/meetingtime.html?month=8&day=22&year=2022&p1=5446&p2=136&p3=87&p4=438&p5=235&iv=1800

Kindly vote or respond to this thread indicating your interest and availability. And any other suggestions you may have regarding scheduling.

**Comment by bokdeuk-jeong** [comment]
Hi Aditya,
I've been waiting for this meeting. :)  It is 12:00am in Korea, but I think I can make time since it's once a month.

**Reply by gapisback** [reply]
Thanks, @bokdeuk-jeong for agreeing to join.

Let me see what the others say. One option will be to do this at 6am PT == 9 am ET (US) == 3pm Geneva == 10 pm your time.

That works for me, but let's see what others who are located in CA PT time zone have to say for that.

**Reply by jinbpark** [reply]
A monthly 12:00am meeting works for me, too. (but it's hard to guarantee though i will always be available, but I'll try attending if possible)
Anyway, glad to see you here Aditya!

**Comment by rgerganov** [comment]
8AM PT works for me. Thanks.

**Comment by yelvmw** [comment]
Most of the time proposed should work for me. Thanks, Aditya!

Ye


> On Apr 2, 2024, at 2:50 AM, Radoslav Gerganov ***@***.***> wrote:
> 
> 
> 8AM PT works for me. Thanks.
> 
> —
> Reply to this email directly, view it on GitHub <https://www.google.com/url?q=https://github.com/ccc-certifier-framework/certifier-framework-for-confidential-computing/discussions/239%23discussioncomment-8979890&source=gmail-imap&ust=1712645461000000&usg=AOvVaw0PnJLd4deZPaL8KQO6tqZS>, or unsubscribe <https://www.google.com/url?q=https://github.com/notifications/unsubscribe-auth/A2FOFZXMWYHFDORBYFP3LNTY3JIM7AVCNFSM6AAAAABFSTRRX2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSNZZHA4TA&source=gmail-imap&ust=1712645461000000&usg=AOvVaw3XVXGBHOOdAfd9VCFRWBvA>.
> You are receiving this because you are subscribed to this thread.
> 


-- 
This electronic communication and the information and any files transmitted 
with it, or attached to it, are confidential and are intended solely for 
the use of the individual or entity to whom it is addressed and may contain 
information that is confidential, legally privileged, protected by privacy 
laws, or otherwise restricted from disclosure to anyone else. If you are 
not the intended recipient or the person responsible for delivering the 
e-mail to the intended recipient, you are hereby notified that any use, 
copying, distributing, dissemination, forwarding, printing, or copying of 
this e-mail is strictly prohibited. If you received this e-mail in error, 
please return the e-mail to the sender, delete it from your computer, and 
destroy any printed copy of it.

**Comment by gapisback** [comment]
(Re-post from a private email ...)

The enthusiastic response from this core team for this proposed meeting has been overwhelming.

I've been working with CCC people and am getting a bit of back-end resources-setup in-place.

You all should have received an meeting invite for:

- Kick-off meeting on Wed 24th April, at 6-7am PT, along with zoom link.

- (Yes, 6am PT. I cajoled a few US colleagues to accomodate an early start-time, which will mean our Samsung colleagues can join at a reasonable start time of 10pm their local time.)

- The meeting schedule only allows me to do recurring meeting for 4th Wed of each month (as opposed to last Wed). So, let's start this 24th. Next one will be on May 22nd.

- **Agenda**: Drop me a note with your thoughts on agenda items. I'm trying to get access to a shared Google drive docs to maintain minutes and notes. Will update you all when that's available.

I have included Chris, John, Ye, Rado, Samsung (Taesso, Bokdeuk, Sangwan, Jinbum) and Larry Dewey (AMD).

Please let me know who else to include in the invitation.

(I figured the Keystone guys, Evgeny and Daeyeol Lee, are probably off of this project in their own jobs, so didn't include them.)

We may have to go thru some bootstrapping hiccups, so let's see how this turns out.

Thanks,
--A>

**Comment by gapisback** [comment]
Thank you all for attending the Certifier Framework kick-off meeting this morning.

As suggested by Jinbum, I went the Wiki-route to log meeting notes. Here are a few things you might find useful.

1. [Certifier-Framework-Monthly-Meeting-Minutes](https://github.com/ccc-certifier-framework/certifier-framework-for-confidential-computing/wiki/Certifier-Framework-Monthly-Meeting-Minutes) You can find all meeting-info in this one place, along with zoom link.
2. [2024‐CFCC‐Meeting‐Minutes](https://github.com/ccc-certifier-framework/certifier-framework-for-confidential-computing/wiki/2024%E2%80%90CFCC%E2%80%90Meeting%E2%80%90Minutes)
3. [Today's meeting minutes, 4-24-2024](https://github.com/ccc-certifier-framework/certifier-framework-for-confidential-computing/wiki/2024%E2%80%9004%E2%80%9024%E2%80%90CFCC%E2%80%90Minutes)
4. [Next meeting on 22.May](https://github.com/ccc-certifier-framework/certifier-framework-for-confidential-computing/wiki/2024%E2%80%9005%E2%80%9022%E2%80%90CFCC%E2%80%90Minutes)

See you all in May. Will follow-up on individual AIs as appropriate.

Closing out this discussion item.
